### DISCUSSION TITLE
Design System vercel open source

### DISCUSSION THREAD

**Discussion by douglaszaltron** [vercel/vercel#6094] [Help] [answered]
Is there any discussion about leaving the design (https://vercel.com/design
) code open as well, I would love to see how it was developed. For the community I think it would be interesting to even build applications on top of it with the vercel or custom theme.

I didn't know where would be the best channel for this question.

**Comment by uneknown** [comment]
I'd love to see that! To be honest, I don't think they'll do it. They do not benefit from doing so.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out! While the general design is open, the UI library is not. But there is a community UI library project based on Vercel's design called [Geist](https://github.com/geist-org/geist-ui). Depending on what you want to accomplish, that unofficial Vercel design clone could be what you're looking for! There are also a number of other open design systems to explore at https://open-ui.org/design-systems

**Reply by douglaszaltron** [reply]
The interesting thing in my view is that the project is open source, that the community could also help in some problems and not so much in using it (as you mentioned there is the Geist) and that alias if this project exists "Geist", was because it had interest in using the design of vercel. And the fact that the company defends this cause, since the design is open, why not the code? It would be beautiful, figma files and codes being explored by the community and being one more project in the company's open list. Thanks for reaching out =)

**Comment by BuchananDevOps** [comment]


**Reply by amyegan** [reply]
Thanks for sharing your thoughts @douglaszaltron! There's a lot here so I'm going to need to take some time to thoroughly read it all, but I do have some notes after a quick read. Forgive me if my brief scan gave me an incorrect understanding of your concern!

Geist UI and NextUI are community projects and are **_not_** maintained by the Vercel team. The [NextUI site](https://nextui.org) is hosted by Vercel because we like to support cool open source projects that make things faster for developers! But the [UI library](https://github.com/nextui-org/nextui) is entirely community built with no developer time or resources dedicated from Vercel. Geist UI is not sponsored or maintained by Vercel.

If you are having trouble with Geist UI or NextUI, you should reach out to their developers directly in [Geist UI Discussions](https://github.com/geist-org/geist-ui/discussions) or [NextUI Discussions](https://github.com/nextui-org/nextui/discussions). I'm sure they want to quickly fix any bugs your seeing, and they know their projects best to be able to give you advice when you run into trouble.

**Reply by BuchananDevOps** [reply]
@amyegan Sorry for the off the cuff grievances. Being a devrel is tough enough. I wasn't looking to actually get technical help with either products. 

A team member had sent over the tweet referenced below last week, with [@modulz/design-system](https://design-system.modulz-deploys.com/) being developed and offered publicly after WorkOS acquired Modulz, we collectively assumed Vercel was gearing up to begin marketing Geist with the social posts. 

> Imbuing our work with care and craft as stewards of the Vercel Brand and Geist Design System.

![gesit-ui-tweet](https://user-images.githubusercontent.com/110317902/202056799-668476aa-313b-4718-814e-8aa8960a6d85.png)
 [rauno (@raunofreiberg) November 4, 2022](https://twitter.com/raunofreiberg/status/1588514114896244736?ref_src=twsrc%5Etfw)

**Comment by DigitalGreyHat** [comment]
@amyegan What CSS-in-JS does the design system use?[ vanilla-extract](https://vanilla-extract.style/)? The class names look like they're generated by [vanilla-extract.](https://vanilla-extract.style/)

**Reply by Manuservus11** [reply]
@amyegan would be interesting to know.
